hack/dockerized

```diff
@@ -109,6 +114,11 @@ if [ -z "$(docker ps --format '{{.Names}}' | grep ${BUILDER}-bazel-server)" ]; t
     docker run --network host -d ${volumes} --security-opt label:disable --name ${BUILDER}-bazel-server -w "/root/go/src/kubevirt.io/kubevirt" --rm ${KUBEVIRT_BUILDER_IMAGE} hack/bazel-server.sh
 fi
 
+# Update cert trust, if custom is provided
+if [ -n "$DOCKER_CA_CERT_FILE" ] && [ -f "$DOCKER_CA_CERT_FILE" ]; then
+    docker exec ${BUILDER}-bazel-server /entrypoint.sh "/usr/bin/update-ca-trust"
```

Comment:
I guess this does not take too long, to be an issue, if it is executed over and over?

Reply:
It doesn't take any noticeable time, not compared to the rsyncs and other things. I'm not sure it is worth adding change detection to, in order to determine if trust store needs an update per custom CA contents.

Reply:
Also might be worth pointing out that the user can set the variable only when they want to push a CA change to the image, and it won't run otherwise without the variable.